src/app.js

```diff
@@ -1,21 +1,43 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
+const http = require('http');
+const fs = require('fs');
+const url = require('url');
+
+const PORT = process.env.PORT || 8080;
+
+function staticServer() {
+  const server = http.createServer((req, res) => {
+    const normalizedURL = new url.URL(req.url, `http://${req.headers.host}`);
+
+    if (normalizedURL.pathname.startsWith('/file')) {
+      const fileName = (normalizedURL.pathname === '/file'
+      || normalizedURL.pathname === '/file/')
+        ? 'index.html'
+        : normalizedURL.pathname.replace('/file/', '');
+
+      fs.readFile(`./public/${fileName}`, (err, data) => {
+        if (!err) {
+          res.end(data);
+        }
+
+        res.statusCode = 404;
+        res.end();
+      });
```

Comment:
```suggestion
        if (err) {
          res.statusCode = 404;
          res.end(
        }

        res.statusCode = 200;
        res.end(data);
      });
```
not a mistake, but i think this approach is more convenient